samples/cs/FoundryLocal-01-MEAI-Chat/Program.cs

```diff
@@ -0,0 +1,28 @@
+﻿using OpenAI;
+using OpenAI.Chat;
+using System.ClientModel;
+using System.Text;
+
+var model = "Phi-3.5-mini-instruct-cuda-gpu";
+var baseUrl = "http://localhost:5273/v1";
+var apiKey = "unused";
+
```

Comment:
Would it be possible to use the Foundry Local C# SDK to get the `baseUrl` and `apiKey` - in a future release, the Port and Key will be dynamically set by foundry on start up.

Example of SDK:

```csharp
var alias = "phi-3.5-mini";

var manager = await FoundryManager.StartModelAsync(aliasOrModelId: alias);

var model = await manager.GetModelInfoAsync(aliasOrModelId: alias);
ApiKeyCredential key = new ApiKeyCredential(manager.ApiKey);
OpenAIClient client = new OpenAIClient(key, new OpenAIClientOptions
{
    Endpoint = manager.Endpoint
});
```

The official NuGet should be release this week. In the meantime, you could build from source to test.